usbsdmux/__main__.py

```diff
@@ -92,6 +94,9 @@ def main():
             elif args.action in ["1", "high"]:
                 ctl.gpio_set_high(args.gpio)
 
+        elif mode == "info":
+            ctl.info()
+
```

Comment:
Should `mode == "info"` maybe also do an implicit `ctl.mode_host()`?

When I first tried it just now the first thing I got was an

    usbsdmux.usb2642.SDTransactionFailed: SCSI Transaction ended with status 2. SD Transaction has probably failed.

Exception. I know the USB-SD-Mux internals, so I knew what went wrong, but it may not be obvious to all.
Alternatively we should catch the exception and add a hint to doing an `usbsdmux … host` first.

Reply:
Perhaps instead internally get the state and print the hint if not in host mode?

Reply:
Read the comment before trying it out, but promptly ran into the same issue :-)
:+1: on the state check.

Reply:
I've added a state check.